lang/python/python3-readline/patches/010-link-with-system-libs.patch

```diff
@@ -0,0 +1,78 @@
+Links dynamically to system's libreadline/libhistory.
+--- a/setup.py
++++ b/setup.py
+@@ -7,11 +7,6 @@ import subprocess
+ 
+ from setuptools import setup, Extension
+ 
+-if sys.platform == 'win32':
```

Comment:
can this happen?
i.e. someone to build this on win32 or cygwin?

if not, i would remove this from the patch, it's noise

Reply:
What???

It's not supposed to link against hosts libraries or support Cygwin, this package is compiled for target architechture that is Openwrt, and as far as I know, Openwrt does not "run" on Cygwin nor windows. It's awhile since I created this, but every patch is there for a reason, and if I remember correctly, this one fixes linking issue and instead links it dynamically, not statically, to reduce size of package.

Reply:
I agree with @commodo.  Even with this code, it should never evaluate as True, so it's harmless...